### PR TITLE
Add getLanguages

### PR DESCRIPTION
**Pull request by fiedsch**
added another missing hook description and corrected a typo in a related file

### CODE REVIEW

File: api/extensions/hooks/getLanguages.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# getLanguages
+
+The `getLanguages` hook allows to modify the system's list of languages.
+
+> **Tag** Available from version 3.3.RC1.
+
+
+## Parameters
+
+$languages, $langsNative, $blnInstalledOnly
```
Comment (aschempp) [inline api/extensions/hooks/getLanguages.md]: I guess this shouldn't be here? :-)

**Comment by aschempp** [comment]
Thanks! Please also update the navigation (`SUMMARY.md`).

**Comment by fiedsch** [comment]
Thanks for your input.

> I guess this shouldn't be here? :-)

you are right :-(

> Thanks! Please also update the navigation (SUMMARY.md).

I did not notice that we have the TOC in 

```
api/extensions/hooks/README.md
```

and also in 

```
api/SUMMARY.md
```

I'll update `SUMMARY.md`. But does it make sense to repeat the contents of `README.md` there?

I'll fix the issues when back at home tonight.

**Comment by fiedsch** [comment]
fixed the PR as proposed

**Pull request merged**